sqtom/tests/test_fitting_2d.py

```diff
@@ -94,5 +98,6 @@ def test_marginal_calcs_2d():
     nmean = 1.0
     ps = twinbeam_pmf({"n_modes": 1.0, "sq_0": nmean})
     assert np.allclose(
-        marginal_calcs_2d(ps, as_dict=False), np.array([nmean, nmean, 2 + 1 / nmean, 2, 2]),
+        marginal_calcs_2d(ps, as_dict=False),
+        np.array([nmean, nmean, 2 + 1 / nmean, 2, 2, 0]),
```

Comment:
(Minor suggestion): for readability, the result and the expected object could be defined prior to calling `np.allclose`, for example:
```python
res = marginal_calcs_2d(ps, as_dict=False)
expected = np.array([nmean, nmean, 2 + 1 / nmean, 2, 2, 0])
assert np.allclose(res, expected)
```